### PR TITLE
Add support for setup in  GPC transport

### PR DESCRIPTION
**Pull request by kilatib**
# Allow GPC topic be created by Symfony 

## Development impact
1. Extended `GpsTransport` with implementing `SetupableTransportInterface` Symfony interface.
2. Added new method `GpsTransport::setup`
3. Added UnitTest for new method
4. Added test for `GpsTransportFactory::support` to validate correct transport attachment. 
Снимок экрана 2022-07-27 в 11 55 52

## Steps to reproduce 
1. Go to the project inside the terminal 
6. Run `bin/console messenger:setup-transports`
7. You will see the message: `! [NOTE] The "{queueName}" transport does not support setup. `   

## Acceptance criteria
1. Go to the project inside the terminal 
2. Run `bin/console messenger:setup-transports`
3. You will see the message: `[OK] The "{queueName}" transport was set up successfully. `   

## Steps for test
1. Create GPS emulator pub/sub
2. Setup new queue by DSN with value: `gps://defaults/{queueName}?client_config[apiEndpoint]=http://{emulatorHost}:{emulatorPort}&max_messages_pull=10`
2.1 {queueName} - future queue name
2.2. {emulatorHost} - host of emulator (default: localhost)
2.3 {emulatorPort} - port of emulator (default: 8432)
3. Add a new queue to config for `packages/messenger.yml`
```YAML
# config/packages/messenger.yaml

framework:
    messenger:
        transports:
        transports:
            messages:
                dsn: 'gps://defaults/messages?client_config[apiEndpoint]=http://localhost:8432&max_messages_pull=10`'
```
8. Go to project terminal and run: `bin/console messenger:setup-transports`
9. Should see the message: `[OK] The "messages" transport was set up successfully. `

### CODE REVIEW

**Comment by marforon** [comment]
@kilatib thank you for your work! This is useful, later we can add also ability to configure options for a created topic/subscription to override defaults.

I had to upgrade phpcs fixer in master because of unsupported version for github workflows. Could you rebase onto latest master? I would like to let it pass CI before we merge it and release it.

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...ble_default_argument_value":true,"no_useless_else":true,"no_useless_return":true,"ordered_imports":{"imports_order":["class","function","const"],"sort_algorithm":"alpha"},"phpdoc_align":{"align":"left"},"phpdoc_order":true,"psr_autoloading":true,"no_unused_imports":true,"semicolon_after_instruction":true,"strict_comparison":true,"strict_param":true},"hashes":{"Tests\/Transport\/GpsTransportTest.php":2253092003,"Transport\/GpsConfiguration.php":796500406,"Transport\/GpsTransportFactory.php":3375715053,"Transport\/GpsTransport.php":1767249949,"Transport\/GpsConfigurationResolverInterface.php":3072373312,"Transport\/GpsReceiver.php":3101780492,"Transport\/GpsSender.php":487225657,"Transport\/GpsConfigurationResolver.php":1411913899,"Transport\/Stamp\/GpsReceivedStamp.php":2391196650,"Transport\/Stamp\/OrderingKeyStamp.php":381212147,"Transport\/GpsConfigurationInterface.php":973459352,"GpsMessengerBundle.php":2630826950,"Tests\/Transport\/GpsSenderTest.php":280903470,"Tests\/Transport\/EnvelopeFactory.php":581157049,"Tests\/Transport\/GpsTransportFactoryTest.php":2202605007,"Tests\/Transport\/GpsReceiverTest.php":1538574135,"DependencyInjection\/GpsMessengerExtension.php":3555245838}}
```
Comment (marforon) [inline .php_cs.cache:1]: ups :), please delete it

Comment (kilatib) [reply .php_cs.cache:1]: ups :)

**Comment by marforon** [comment]
@kilatib thank you, released in 1.2.0

**Pull request merged**